src/Administration/Resources/app/administration/src/app/component/grid/sw-grid/sw-grid.scss

```
@@ -75,41 +52,41 @@ $sw-grid-table-row-color-background: $color-white;
         grid-template-rows: auto;
 
         &.sw-grid__content--header {
-            grid-template-rows: 64px 1fr;
+            grid-template-rows: var(--scale-size-64) 1fr;
         }
 
         &.sw-grid__content--pagination {
-            grid-template-rows: 1fr 64px;
+            grid-template-rows: 1fr var(--scale-size-64);
         }
 
         &.sw-grid__content--header.sw-grid__content--pagination {
-            grid-template-rows: 64px 1fr auto;
+            grid-template-rows: var(--scale-size-64) 1fr auto;
         }
     }
 
     .sw-grid__header {
         display: grid;
         grid-auto-flow: column;
-        background-color: $sw-grid-header-color-background;
-        border-bottom: 1px solid $sw-grid-header-color-border;
+        background: var(--color-elevation-surface-raised);
+        border-bottom: 1px solid var(--color-border-secondary-default);
 
         .sw-grid-column {
-            font-size: $font-size-s;
-            font-weight: bold;
+            font-size: var(--font-size-s);
```

```suggestion
            font-size: var(--font-size-xs);
```

Not your fault. But should look better like this 😄